homeassistant/components/mqtt/__init__.py

```diff
@@ -28,7 +28,11 @@
 from homeassistant.exceptions import TemplateError, Unauthorized
 from homeassistant.helpers import config_validation as cv, event, template
 from homeassistant.helpers.device_registry import DeviceEntry
-from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.dispatcher import async_dispatcher_connect, dispatcher_send
```

Comment:
```suggestion
from homeassistant.helpers.dispatcher import (
    async_dispatcher_connect,
    async_dispatcher_send,
)
```

Reply:
Why this change, don't we let `isort` manage the formatting of imports?